types/braintree-web/venmo.d.ts

```diff
@@ -98,6 +98,10 @@ export function create(options: {
     ignoreHistoryChanges?: boolean | undefined;
     profileId?: string | undefined;
     deepLinkReturnUrl?: string | undefined;
+    allowDesktopWebLogin?: boolean | undefined;
+    allowDesktop?: boolean | undefined;
+    mobileWebFallBack?: boolean | undefined;
+    paymentMethodUsage?: string | undefined;
```

Comment:
Please make the type of `paymentMethodUsage` stricter, see the possible options in https://braintree.github.io/braintree-web/current/module-braintree-web_venmo.html#.create

Reply:
Update the types paymentMethodUsage Property.